File: 프로그래머스/2/148652. 유사 칸토어 비트열/유사 칸토어 비트열.java
class Solution {
    public int solution(int n, long l, long r) {
        int answer = 0;
        for(long i=l-1; i<r; i++){
            boolean isOne = true;
            long num = i;
            while(num >=5){
                if(num % 5 == 2){
                    isOne = false;
                    break;
                }
                num /= 5;
            }
            if(num == 2) isOne = false;
            if(isOne) answer++;
        }
        return answer;
    }
}

File: 프로그래머스/2/148652. 유사 칸토어 비트열/유사 칸토어 비트열.js
function solution(n, l, r) {
    var answer = 0;
    for(let i=l-1; i<r; i++){
        var isOne = true;
        var num = i;
        while(num >= 5){
            if(num%5 == 2){
                isOne = false;
                break;
            }
            num = Math.floor(num/5);
        }
        if(num == 2) isOne = false;
        if(isOne) answer++;
    }
    return answer;
}